app/screens/post_options/post_options.js

```diff
@@ -398,6 +402,10 @@ export default class PostOptions extends PureComponent {
     render() {
         const {deviceHeight} = this.props;
         const options = this.getPostOptions();
+        if (!options || !options.length) {
+            return null;
+        }
```

Comment:
Not related to issue but added so it's obvious when we do testing that nothing will be rendered if no option is to be rendered.